audit/src/handle.rs

```diff
@@ -121,4 +121,27 @@ impl Handle {
         req.header.flags = NLM_F_REQUEST | NLM_F_ACK;
         self.acked_request(req).await
     }
+
+    /// Get current audit status
+    pub async fn get_status(&mut self) -> Result<StatusMessage, Error> {
+        let mut req = NetlinkMessage::from(AuditMessage::GetStatus(None));
+        req.header.flags = NLM_F_REQUEST | NLM_F_DUMP;
+
+        let mut request = self.request(req)?;
+        if let Some(response) = request.next().await {
```

Comment:
I think you can do `let response = request.next().await.ok_or(Error::RequestFailed)?` and get rid of the `if let`